test/scripts/aws/run-e2e-tests.sh

```diff
@@ -134,5 +134,5 @@ popd
 
 echo "Starting E2E functional tests ..."
 pushd test/e2e >/dev/null
-  pytest -n 4 --ignore=credentials/test_set_creds.py
-popd
+  pytest -n 4 predictor/test_autoscaling.py
```

Comment:
need to change back to run all the tests?

Reply:
I can remove it, but thats no longer used, its under `aws` folder, we use the one under `gh-actions`...